arch/xtensa/src/esp32/esp32_allocateheap.c

```diff
@@ -130,6 +132,10 @@
 void up_allocate_heap(FAR void **heap_start, size_t *heap_size)
 {
   board_autoled_on(LED_HEAPALLOCATE);
+#ifdef CONFIG_XTENSA_IMEM_REGION_SIZE_MAX
+  *heap_size = 0x2000;
```

Comment:
hi @cwespressif please avoid using this hard-coded value (0x2000). Please create a macro symbol to represent this value. This value is used in more than one place, then defining it will simplify case someone want to modify it. Other (better) option is adding it as an option config and setup the minimum and maximum range the user can choose.

Reply:
Thanks for your suggestion, I have modified it.